README.md

```diff
@@ -205,11 +179,11 @@ Example:
 
 > Mention all of the collaborators of this project.
 
```

Comment:
- [Optional] Dear @Reggeditt, Please remove all the `> suggestions` in your readme, This is just to tell you what to do and should not be part of your readme file.